src/main/java/io/opensergo/OpensergoClientStatus.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022, OpenSergo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.opensergo;
+
+/**
+ * @author Jiangnan Jia
+ **/
+public enum OpensergoClientStatus {
```

Comment:
Opensergo -> OpenSergo

Reply:
okay, and i will check the other’s name.

Reply:
finished.